src/components/Project/Project.stories.tsx

```diff
@@ -1,14 +1,14 @@
-import styled from "styled-components";
-import Panel from "../Panel";
+// import styled from "styled-components";
+// import Panel from "../Panel";
```

Comment:
commented out unused imports for now to suppress baby lints

Reply:
you can remove them if it is unused